src/util/tracking.js

```diff
@@ -172,7 +172,7 @@ export const logTrack = platformAgent => (response, trackInfo) => {
 	const eventDate = new Date();
 	const record = {
 		timestamp: eventDate.getTime().toString(),
-		requestId: uuid.v4(),
+		requestId: response.request.id,
```

Comment:
yeah, the Hapi request always had a unique ID, but it wasn't formatted like a UUID, which I think is why we were generating a unique UUID in this step previously. Now, the `onRequestExtension` will overwrite the default `request.id` with a UUID, which can be used consistently for tracking and general logging.

I did some investigation about whether it is 'okay' to override the default `request.id` and everything I saw suggested that it shouldn't be an issue.